src/hotspot/share/gc/shared/pretouchTask.cpp

```diff
@@ -64,13 +64,14 @@ void PretouchTask::work(uint worker_id) {
 
 void PretouchTask::pretouch(const char* task_name, char* start_address, char* end_address,
                             size_t page_size, WorkGang* pretouch_gang) {
-
+  // Chunk size should be at least (unmodified) page size as using multiple threads
+  // pretouch on a single chunk can decrease performance.
```

Comment:
it should actually read "... pretouch on a single page can ..." not chunk :( Sorry, my fault.

Reply:
fixed